### ISSUE TITLE
No oAuth block in amplifyconfiguration.json when importing existing Cognito User Pool

### ISSUE THREAD

**Issue by OzyOzk** [aws-amplify/amplify-cli#8938]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

curl

### If applicable, what version of Node.js are you using?

16.13.0

### Amplify CLI Version

7.3.6

### What operating system are you using?

Windows

### Amplify Categories

auth

### Amplify Commands

Not applicable

### Describe the bug

When I import an existing userpool, using the userpool only option, no oauth key is added to the Auth/Default key. I get the following instead;

```
const amplifyconfig = ''' {
    "auth": {
        "plugins": {
            "awsCognitoAuthPlugin": {
                "UserAgent": "aws-amplify-cli/0.1.0",
                "Version": "0.1.0",
                "IdentityManager": {
                    "Default": {}
                },
                "CognitoUserPool": {
                    "Default": {
                        "PoolId": "eu-west-1_xxxxx",
                        "AppClientId": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
                        "Region": "eu-west-1"
                    }
                },
                "Auth": {
                    "Default": {
                        "authenticationFlowType": "USER_SRP_AUTH",
                        "socialProviders": [],
                        "usernameAttributes": [],
                        "signupAttributes": [
                            "EMAIL"
                        ],
                        "passwordProtectionSettings": {
                            "passwordPolicyMinLength": 8,
                            "passwordPolicyCharacters": [
                                "REQUIRES_LOWERCASE",
                                "REQUIRES_UPPERCASE",
                                "REQUIRES_NUMBERS",
                                "REQUIRES_SYMBOLS"
                            ]
                        },
                        "mfaConfiguration": "OFF",
                        "mfaTypes": [],
                        "verificationMechanisms": [
                            "EMAIL"
                        ]
                    }
                }
            }
        }
    }
}''';
```

I've also tried manually adding in the file, as shown [here](https://docs.amplify.aws/lib/auth/existing-resources/q/platform/ios/) but then I just get  the following error;

`Unhandled Exception: AmplifyException(message: Failed to instantiate AWSMobileClient, recoverySuggestion: See attached exception for more details, underlyingException: java.lang.RuntimeException: Failed to initialize OAuth, please check your awsconfiguration.json)`

### Expected behavior

I was expecting an amplify configuration file in the format shown [here](https://docs.amplify.aws/lib/auth/existing-resources/q/platform/ios/).

When I manually created the file, I was expecting a hosted UI to show once I clicked on my login button.

### Reproduction steps

`ampliy import auth`

select user pool only

`amplify push`

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line

I/amplify:flutter:api( 7636): Added API plugin
I/amplify:flutter:auth_cognito( 7636): Added Auth plugin
D/AWSMobileClient( 7636): Using the SignInProviderConfig from `awsconfiguration.json`.
D/AWSMobileClient( 7636): initialize: Cognito HostedUI client detected
D/AWSMobileClient( 7636): Inspecting user state details
W/InternalCallback( 7636): Library attempted to call user callback twice, expected only once
E/flutter ( 7636): [ERROR:flutter/lib/ui/ui_dart_state.cc(199)] Unhandled Exception: AmplifyException(message: Failed to instantiate AWSMobileClient, recoverySuggestion: See attached exception for more details, underlyingException: java.lang.RuntimeException: Failed to initialize OAuth, please check your awsconfiguration.json)
E/flutter ( 7636): #0      AmplifyClass.configure (package:amplify_flutter/amplify.dart:173:9)
E/flutter ( 7636): <asynchronous suspension>
E/flutter ( 7636): #1      configureAmplify (package:test_amplify/amplify.dart:20:5)
E/flutter ( 7636): <asynchronous suspension>
E/flutter ( 7636): 

```

</details>

### Additional information

My configureAmplify function that throws the rror;

```
void configureAmplify() async {

  // Add Pinpoint and Cognito Plugins, or any other plugins you want to use
  //AmplifyAnalyticsPinpoint analyticsPlugin = AmplifyAnalyticsPinpoint();
  await Amplify.addPlugins([AmplifyAPI(), AmplifyAuthCognito()]);

  // Once Plugins are added, configure Amplify
  // Note: Amplify can only be configured once.
  try {
    await Amplify.configure(amplifyconfig);
  } on AmplifyAlreadyConfiguredException {
    print("Tried to reconfigure Amplify; this can occur when your app restarts on Android.");
  }
}
```

I'm using android studios.

**Comment by OzyOzk** [comment]
For what it's worth, I also cant import auth for amplify infrastructure that were created with the amplify CLI. It only works on the project that I created it in, but if I try starting a new project and doing amplify pull --appId XXXXXX --envName dev, I get the following error;

`Unhandled Exception: AmplifyException(message: Failed to instantiate AWSMobileClient, recoverySuggestion: See attached exception for more details, underlyingException: java.lang.RuntimeException: Failed to initialize OAuth, please check your awsconfiguration.json)`

I tried reverting aws_amplify from 0.2.0 to 0.1.0, but the issue persisted.

**Comment by attilah** [comment]
@OzyOzk I tried to reproduce the same issue by creating a project adding auth with OAuth configuration and pushed. After checking `app/src/main/res/raw/amplifyconfiguration.json` I had the `OAuth` section in the file. Could you check if in the original project where you created the user pool you have this section?

Then imported the same user pool into another empty project pushed and the file content was the same and it had an `OAuth` section.

Did you pick the right appclient during `auth import` ?

**Comment by attilah** [comment]
Closing as no repro, if the issue happens please open a new issue.
